orchagent/port.h

```diff
@@ -64,6 +65,7 @@ class Port
     Type                m_type;
     int                 m_index = 0;    // PHY_PORT: index
     int                 m_ifindex = 0;
+    uint32_t            m_if_mtu = DEFAULT_IF_MTU;
```

Comment:
using m_ip_mtu instead of m_if_mtu will be more clear.

Reply:
The if_mtu is defined in the following schema
https://github.com/Azure/sonic-swss/blob/d1fdf399e1e0e2e0d71fad1bce2ea7babee2eb48/doc/swss-schema.md

So I'm trying to keep it consistent from intfsyncd->app_db->intfsorch->sai by using "if_mtu" instead of "ip_mtu".
Please let me know if you think it's better to change "if_mtu" to "ip_mtu" in both swss schema and everywhere else.